package/MDAnalysis/core/universe.py

```diff
@@ -213,6 +213,10 @@ class Universe(object):
         Universe to only unpickle if a compatible Universe with matching
         *anchor_name* is found. Even if *anchor_name* is set *is_anchor* will
         still be honored when unpickling.
+    transformations: list, optional
+        Provide a list of transformations that you wish to apply to the 
+        trajectory upon reading. Some transformations can be found in 
```

Comment:
Based on the structure of the sentence, it should not be "Some transformations can be found in" but "Transformations can be found in".